bazel.el

```diff
@@ -88,8 +88,17 @@
           "https://github.com/bazelbuild/buildtools/tree/master/buildifier")
   :risky t)
 
-;;;; Commands to run Buildifier.
+(defvar bazel--magic-comment-rx
+  '(or "@unused"
+       "@unsorted-dict-items"
+       "keep sorted"
+       "do not sort"
+       "buildifier: leave-alone"
+       (seq (or "buildifier" "buildozer") ": "
+             (seq "disable=" (+ (any "A-Za-z-")))))
+  "Regular expression used to identify magic comments known to `buildifier'")
```

Comment:
nit: `` `buildifier' `` would refer to some Lisp symbol; prefer writing `Buildifier` here to refer to the binary.

Reply:
Sure thing.  Also linked to the `buildifier` documentation and sources where appropriate.  Please take a look.